packages/prosemirror-lwdita/src/app-config.ts

```diff
@@ -15,15 +15,42 @@ You should have received a copy of the GNU Affero General Public License
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 */
 
+/**
+ * The URL of the Petal frontend and API
+ */
+export const serverConfig = {
+  frontendUrl: process.env.PARCEL_FRONTEND_URL || 'http://localhost:1234/',
+  apiUrl: process.env.PARCEL_API_URL || 'http://localhost:3000/',
+}
+
 export const clientID: { id: string, value: string } = {
   id: 'client_id',
-  value: 'Iv23li0Pvl3E4crXIBQ0',
+  value: process.env.GITHUB_CLIENT_ID || 'Iv23li0Pvl3E4crXIBQ0',
```

Comment:
We should remove the default value, is it's not passed as an environment variable then something is wrong.